internal/handler/web/studies.go

```diff
@@ -60,6 +60,57 @@ func (h *Handler) GetStudies(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, response)
 }
 
+func (h *Handler) GetStudyStudyId(ctx *gin.Context, studyId string) {
+	user := middleware.GetUser(ctx)
+
+	studyUUID, err := uuid.Parse(studyId)
+	if err != nil {
+		setError(ctx, types.NewErrInvalidObject(err), "Invalid study ID format")
+		return
+	}
+
+	study, err := h.studies.StudyWithOwner(studyUUID, user)
+	if err != nil {
+		setError(ctx, err, "Failed to retrieve study")
+		return
+	}
+
+	ownerUserIDStr := study.OwnerUserID.String()
+
+	response := openapi.Study{
```

Comment:
this is a little copy-pasta for my liking – maybe a `studyToOpenApiStudy` func here?

Reply:
https://github.com/ucl-arc-tre/portal/pull/213/commits/5150900d40e86299649ff14eb88d1084b12e0dc5